lib/modules/datasource/custom/formats/plain.ts

```diff
@@ -20,11 +20,6 @@ export class PlainFetcher implements CustomDatasourceFetcher {
   async fetch(http: Http, registryURL: string): Promise<unknown> {
     const response = await http.getPlain(registryURL);
 
-    const contentType = response.headers['content-type'];
-    if (!contentType?.startsWith('text/')) {
-      return null;
-    }
-
```

Comment:
shouldn't we check for string? otherwise we try to parse blobs and that maybe throws somewhere.

Reply:
Based on the types getPlain only can return strings. 
So if we need to add checks, IMO it should be there.